docusaurus.config.js

```diff
@@ -12,10 +12,17 @@ function getAlgoliaConfig(
   requiredKeys = ['appId', 'apiKey', 'indexName']
 ) {
   // filter missing required keys
-  const missingKeys = requiredKeys.filter((key) => config[key] === undefined)
+  const missingKeys = requiredKeys.filter(
+    (key) => config[key] === undefined || config[key] === ''
+  )
 
   // if some keys are missing, return undefined
   if (missingKeys.length > 0) {
+    // require all keys in production
+    if (process.env.NODE_ENV == 'production') {
```

Comment:
`process.env.NODE_ENV === 'production'`

Reply:
idk if this is the problem though

Reply:
I think the issue is `NODE_ENV` is set to 'production' when doing `npm run build`. Should we just use some other variable name?

Reply:
yeah I think we should prob not overcomplicate this and just change the name